src/Pages/Tvshows/Tvshowid_Int.elm

```diff
@@ -56,16 +58,40 @@ init shared { params } =
     ( { tvshowid = params.tvshowid, tvshow = getTvShow params.tvshowid shared.tvshow_list, tvshow_list = shared.tvshow_list, season_list = shared.season_list, episode_list = shared.episode_list }
     , sendActions
         [ """{"jsonrpc": "2.0", "method": "VideoLibrary.GetSeasons", "params": {"tvshowid": """ ++ String.fromInt params.tvshowid ++ """ ,"properties":["season","episode","tvshowid","art","watchedepisodes"]}, "id": "libSeasons"}"""
-        , """{"jsonrpc": "2.0", "method": "VideoLibrary.GetEpisodes", "params": {"tvshowid": """ ++ String.fromInt params.tvshowid ++ """ ,"properties":["tvshowid","seasonid","episode","title","art","season"]}, "id": "libEpisodes"}"""
+        , """{"jsonrpc": "2.0", "method": "VideoLibrary.GetEpisodes", "params": {"tvshowid": """ ++ String.fromInt params.tvshowid ++ """ ,"properties":["tvshowid","seasonid","episode","title","art","season","file","firstaired","director","writer","cast","streamdetails","plot","rating","runtime"]}, "id": "libEpisodes"}"""
```

Comment:
We shouldn't call it here. This should be the responsibility of the seasons page.

Reply:
Hi @razzeee, 
The reason we are calling all episodes for a TvShow on this page is that we need an `episode_list` in order to queue/play all of the episodes in the TvShow when the play/queue button is clicked. If we call it from the season's page then on the `tvshowid_int` page the `episode_list` will remain empty thus not enabling functions like playing and queuing the whole tvshow.

Reply:
Okay, still, we should fetch on the season Page again